Sources/Core/GTMSessionFetcher.m

```diff
@@ -2598,9 +2603,14 @@ - (void)invokeOnCallbackUnsynchronizedQueueAfterUserStopped:(BOOL)afterStopped
 - (void)invokeOnCallbackQueue:(dispatch_queue_t)callbackQueue
              afterUserStopped:(BOOL)afterStopped
                         block:(void (^)(void))block {
+
   if (callbackQueue) {
     dispatch_group_async(_callbackGroup, callbackQueue, ^{
-      if (!afterStopped) {
+      BOOL skipEarlyExit = NO;
+      @synchronized(self) {
```

Comment:
this sync isn't needed now that it can't change after begin.

Reply:
removed